docs/index.html

```diff
@@ -45,7 +45,12 @@ <h3>Get up and running <em>in&nbsp;seconds</em>.</h3>
         <p class="line">
           <span class="path">~</span>
           <span class="prompt">$</span>
-          <span class="command">gem install jekyll bundler</span>
+          <span class="command">gem install bundler</span>
+        </p>
+        <p class="line">
+          <span class="path">~</span>
+          <span class="prompt">$</span>
+          <span class="command">bundle add jekyll &amp; &amp; bundle install</span>
```

Comment:
I'm...not that clever. Sorry about this - as written, it's a syntax error. Fix is in #6969.